### DISCUSSION TITLE
shapenet dataset raw

### DISCUSSION THREAD

**Discussion by spyroot** [pyg-team/pytorch_geometric#3447] [Q&A] [answered]
Hi Folks,

Do you know what each column in Shapenet Raw?
I can't find anywhere, and I see you are only parsing three columns ( 4, 5, 6 )
-0.261980 -0.006220 -0.213330 -0.592600 0.268700 -0.759300 6.000000

Thank you very much.

**Comment by rusty1s** [comment, marked as answer]
The first 3 entries denote the `xyz` coordinates of the point.
The next 3 entries denote the normal vector of each point.
The last entry denotes the label.

**Reply by spyroot** [reply]
Thank you very much.  !
